addons/sale/wizard/sale_order_discount.py

```diff
@@ -134,7 +135,7 @@ def _create_discount_lines(self):
                         description=_(
                             "Discount: %(percent)s%%"
                             "- On products with the following taxes %(taxes)s",
-                            percent=self.discount_percentage*100,
+                            percent=round(self.discount_percentage*100, discount_dp),
```

Comment:
```suggestion
                            percent=float_repr(self.discount_percentage * 100, discount_dp),
```

Same as above. Also might as well add some spaces around the operator when modifying these lines.